tests/f219.rs

```diff
@@ -0,0 +1,85 @@
+use hayagriva::citationberg::IndependentStyle;
```

Comment:
I do wonder if this could become a `local` citeproc test with the new bibliography support.

Though it'd generally be nice to have better infrastructure for non-citeproc (hayagriva YAML) tests. I wonder if we could have a nice little macro which does all of this in a less repetitive way, so we can have multiple tests in a single file.